modules/geo-layers/src/tile-layer/tile-layer.js

```diff
@@ -3,22 +3,22 @@ import {GeoJsonLayer} from '@deck.gl/layers';
 import TileCache from './utils/tile-cache';
 
 const defaultProps = {
-  renderSubLayers: props => new GeoJsonLayer(props),
-  getTileData: ({x, y, z}) => Promise.resolve(null),
-  onDataLoaded: () => {},
+  renderSubLayers: {type: 'function', value: props => new GeoJsonLayer(props)},
+  getTileData: {type: 'function', value: ({x, y, z}) => Promise.resolve(null)},
+  onViewportLoaded: {type: 'function', value: () => {}},
   // eslint-disable-next-line
-  onGetTileDataError: err => console.error(err),
-  maxZoom: null,
-  minZoom: null,
-  maxCacheSize: null
+  onTileError: {type: 'function', value: err => console.error(err)},
+  maxZoom: {min: 0, value: null},
+  minZoom: {min: 0, value: 0},
```

Comment:
- Type must be specified if default is an object.
- There should not be a min value.

Just use `maxZoom: null, minZoom: 0`